server/test/BraveAlerterConfiguratorTest.js

```diff
@@ -1,280 +1,280 @@
+/* eslint-disable import/no-extraneous-dependencies */
 const { ALERT_STATE, AlertSession } = require('brave-alert-lib')
 const chai = require('chai')
+
 const expect = chai.expect
 const { afterEach, before, beforeEach, describe, it } = require('mocha')
 const sinon = require('sinon')
-const sinonChai = require("sinon-chai")
+const sinonChai = require('sinon-chai')
 const db = require('../db/db.js')
 const SessionState = require('../SessionState.js')
 
-const BraveAlerterConfigurator = require('./../BraveAlerterConfigurator.js')
+const BraveAlerterConfigurator = require('../BraveAlerterConfigurator.js')
```

Comment:
Funny, when I ran the linter/prettier in `brave-alert-lib`, it made me remove all the `.js` extensions from lines like this.

Reply:
that is very strange and I'm not sure why it would do that inconsistently but I haven't manually touched anything that should affect that